docs/diagrams/ModelClassDiagram.puml

```diff
@@ -5,23 +5,23 @@ skinparam arrowColor MODEL_COLOR
 skinparam classBackgroundColor MODEL_COLOR
 
 Package Model <<Rectangle>>{
-Interface ReadOnlyAddressBook <<Interface>>
+Interface ReadOnlyRecipeBook <<Interface>>
 Interface Model <<Interface>>
 Interface ObservableList <<Interface>>
-Class AddressBook
-Class ReadOnlyAddressBook
+Class RecipeBook
+Class ReadOnlyRecipeBook
 Class Model
 Class ModelManager
 Class UserPrefs
 Class ReadOnlyUserPrefs
 
-Package Person {
-Class Person
-Class Address
-Class Email
+Package Recipe {
+Class Recipe
+Class Instruction
```

Comment:
should we use IngredientList or Ingredient? 
same with the instruction? 
pls check with the rest.

Reply:
i think it should be IngredientList and InstructionList

Reply:
I think it should be IngredientList and InstructionList